en/docs/learn/consume-api/customizations/customizing-the-developer-portal/customize-api-listing/categorizing-and-grouping-apis/api-category-based-grouping.md

```diff
@@ -0,0 +1,86 @@
+
+# API Category based Grouping
+
+You can use API categories to group APIs. In previous versions of WSO2 API Manager, the process of grouping APIs was carried out by using Tagwise groups. 
+
+!!! note
+
+    The [Tagwise grouping feature]({{base_path}}/learn/consume-api/customizations/customizing-the-developer-portal/customize-api-listing/categorizing-and-grouping-apis/tagwise-based-grouping) has been deprecated with the introduction of the API categories feature in WSO2 API Manager 3.1.0. Therfore, WSO2 recommends all users to use API categories instead of Tagwise grouping.
+
+Initially, the Admins will define API categories. Thereafter, API providers will add API categories to APIs when designing them via the API Publisher. API categories allow API providers to categorize APIs that have similar attributes. When a categorized API gets published to the Developer Portal, its categories appear as clickable links to the API consumers. The API consumers can use the available API categories to quickly jump to a category of interest.
+
+## Step 1 - Add an API Category
+
+You can add an API category using any of the following methods:
+
+- [Add API Category using the Admin Portal UI]({{base_path}}/learn/consume-api/customizations/customizing-the-developer-portal/customize-api-listing/categorizing-and-grouping-apis/api-category-based-grouping/#add-api-category-using-the-admin-portal-ui)
+- [Add API Category using the Admin REST API]({{base_path}}/learn/consume-api/customizations/customizing-the-developer-portal/customize-api-listing/categorizing-and-grouping-apis/api-category-based-grouping/#add-api-category-using-the-admin-portal-ui)
```

Comment:
this should be corrected to 
`[Add API Category using the Admin REST API]({{base_path}}/learn/consume-api/customizations/customizing-the-developer-portal/customize-api-listing/categorizing-and-grouping-apis/api-category-based-grouping/#add--an-api-category-using-the-admin-rest-api)` ?

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/933/commits/71b90577c26a2b9418ad0170c7aace8a7b47d9a3